src/sandstorm/api-session.capnp

```diff
@@ -30,7 +30,103 @@ interface ApiSession @0xc879e379c625cdc7 extends(WebSession.WebSession) {
     # it.  However, for those that benefit from it (like analytics), clients can opt into passing
     # their IP on to the backend by adding an "X-Sandstorm-Passthrough: address" header to their
     # request.  This would be a privacy leak for WebSession, since the grain can give the client
-    # scripts which would send the header, but ApiSession requires a user action, so it's safe here.
+    # scripts which would send the header, but ApiSession requires a user action, so it's safe
+    # here.
     remoteAddress @0 :IpAddress;
   }
+
+  struct PowerboxTag {
+    # A tag used for PowerboxDescriptors describing HTTP APIs. Such descriptors should typically
+    # indicate `ApiSession` as the requested interface and use HttpApiTag to specify what API is
+    # expected.
```

Comment:
So the ID of `PowerboxTag` is meant to be used in `PowerboxDescriptor.Tag.id`, with a value of type `PowerboxTag` in `PowerboxDescriptor.Tag.value`?